test-integration/scripts/10-install-jhipster.sh

```diff
@@ -49,7 +49,9 @@ if [[ "$JHI_REPO" == *"/generator-jhipster" ]]; then
     git --no-pager log -n 10 --graph --pretty='%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit
 
     npm ci
-    npm install -g "$JHI_HOME"
+    npm pack
```

Comment:
After I switched to `npm-shrinkwrap.json`, I can no longer install from the directory. To fix CI, I switched to the packaged approach as that solves the issue and also simulates the published package. If anyone has a better solution, then, kindly let me know.

Reply:
I think it will impact all the other CI, like:
- jhipster lib
- jhipster-vuejs
- jhipster-kotlin
- jhipster-daily-build

I need to have a look